homeassistant/components/google_generative_ai_conversation/strings.json

```diff
@@ -1,17 +1,24 @@
 {
   "config": {
     "step": {
-      "user": {
+      "api": {
         "data": {
-          "api_key": "[%key:common::config_flow::data::api_key%]",
-          "llm_hass_api": "[%key:common::config_flow::data::llm_hass_api%]"
-        }
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "description": "Get an API key from https://aistudio.google.com/app/apikey"
```

Comment:
Inject the url als a description placeholder and use markdown to make the link attached to text